### PR TITLE
T12424: add webkit

### PR DESCRIPTION
**Pull request by reymondzzzz**
Specify library name and version:  **lib/1.0**

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by worldemar — CHANGES_REQUESTED** [review]
Дождаться проверок и переименовать.

**Review by worldemar — APPROVED** [review]

File: recipes/qtwebkit/5.212.0-alpha4.dssl1/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        "libwebp/1.1.0",
+        "sqlite3/3.31.0",
+        "icu/64.2",
+        "libxml2/2.9.9",
+        "libxslt/1.1.33",
+        "zlib/1.2.11"
+    ]
+
+    if platform.system() == "Linux":
+        requires.append("libxcomposite/0.4.5")
```
Comment (Artalus) [inline recipes/qtwebkit/5.212.0-alpha4.dssl1/conanfile.py:56]: это должно быть в `requirements(self)`

File: trassir/lnvr-gcc8/conanfile.txt [hunk 2]
```diff
@@ -117,10 +105,90 @@ qt:qtx11extras=True
...
+expat:shared=True
+grpc:shared=True
+libalsa:shared=True
+libcurl:shared=True
+dlib:shared=header_only
+facesdk:shared=True
+ffmpeg:shared=True
+freetype:fPIC=True
+freetype:shared=True
+freetype:with_bzip2=True
```
Comment (Artalus) [inline trassir/lnvr-gcc8/conanfile.txt:136]: здесь ниже разделить `shared` и остальные флаги, чтобы не возникало ощущения что `opencv:shared=False`

**Review by Artalus — CHANGES_REQUESTED** [review]

File: recipes/qtwebkit/5.212.0-alpha4.dssl1/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                self.build_requires("flex/2.6.4")
+
+        if not tools.which("gperf"):
+            self.build_requires("gperf/3.1")
+        if not tools.which("ruby"):
+            self.build_requires("ruby/2.3.7")
+
+    def requirements(self):
+        if platform.system() == "Linux":
+            requires.append("libxcomposite/0.4.5")
```
Comment (Artalus) [inline recipes/qtwebkit/5.212.0-alpha4.dssl1/conanfile.py:72]: `self.requires`

**Review by Artalus — CHANGES_REQUESTED** [review]

**Review by Artalus — APPROVED** [review]

**Pull request merged**